.travis.yml

```diff
@@ -20,10 +21,17 @@ script:
   - set -o pipefail
   - xcodebuild -version
   - xcodebuild -showsdks
+  - if [[ ${GREY_RUBY} ]]; then
+      cd gem;
+      bundle install --retry=3;
```

Comment:
Do we know what why `bundle install` would require retries? Is it common that git & network requests would fail?